addons/block_code/ui/blocks/block/block.gd

```diff
@@ -217,10 +217,11 @@ func _get_tooltip(at_position: Vector2) -> String:
 	if not definition:
 		return ""
 
+	var description_tx := tr(definition.description)
 	if definition.variant_type == Variant.Type.TYPE_NIL:
-		return definition.description
+		return description_tx
 
-	return "{description}\n\nType: [b]{type}[/b]".format({"description": definition.description, "type": type_string(definition.variant_type)})
+	return "{description}\n\nType: [b]{type}[/b]".format({"description": description_tx, "type": type_string(definition.variant_type)})
```

Comment:
We'll want to translate the string “Type: ” in here. Perhaps this should be broken up a little...

Reply:
Ah, good catch. I didn't even notice it in there. I think I can wrap the whole thing in `tr` and it'll work out like shown [here](https://docs.godotengine.org/en/stable/tutorials/i18n/internationalizing_games.html#placeholders).

Reply:
Yeah we can but it seems a little unkind (and error prone) to translators to make them preserve the format string. Also if we later wanted to lay the tooltip out differently, e.g. using some kind of grid to present attributes in a table, we would want a standalone translation of `Type:`.

Personally I would mark `Type:` for translation and interpolate that into the final string.

Reply:
Yeah, that makes sense. I'll fix that up.